lib/puppet/util/metric.rb

```diff
@@ -26,7 +26,7 @@ def to_data_hash
     {
       'name' => @name,
       'label' => @label,
-      'values' => @values
+      'values' => @values.map { |val| "%0.5f" % val if val.is_a?(Float) }
```

Comment:
@joshcooper here?

Reply:
Oops, I didn't see your suggestion until now. Which would you prefer?

Reply:
yeah, though `val` is an array where each element looks like:

```
["schedule", "Schedule", 0.00027199999999999994]
```

We also want to preserve the float value so it comes out as a number, not a string, so just `val[2].round(N)`. I'm curious at what point JSON switches to exponential notation?

Reply:
Oh, I did not realize that's what `val` was in this case. Okay, I'll update.